lib/presentation/atoms/list_tile_atom.dart

```diff
@@ -21,19 +24,46 @@ class ListTileAtom extends StatelessWidget {
   final bool translateSubtitle;
   final VoidCallback? onTap;
   final bool enable;
+  final ListTileVariant variant;
+  final bool isCrossed;
 
   @override
   Widget build(BuildContext context) {
-    return ListTile(
-      enabled: enable,
-      contentPadding: EdgeInsets.zero,
-      title: TextAtom(title, translate: translateTitle),
-      subtitle: subtitle == null
-          ? null
-          : TextAtom(subtitle!, translate: translateSubtitle),
-      leading: leading,
-      trailing: trailing,
-      onTap: onTap,
-    );
+    switch (variant) {
+      case ListTileVariant.regular:
+        return ListTile(
```

Comment:
There is no reason to duplicate the code for each case.
Make var of the subtitle and inset it in the end

Reply:
Please elaborate, I am unable to understand what you mean

Reply:
```

  @override
  Widget build(BuildContext context) {

  Widget title ;
    switch (variant) {
      case ListTileVariant.regular:
        title = TextAtom(title, translate: translateTitle)
      case ListTileVariant.strikethrough:
        title = AnimatedLineThrough(
          duration: const Duration(milliseconds: 300),
          strokeWidth: 2,
          isCrossed: isCrossed,
          child: TextAtom(title, translate: translateTitle),
        );
    }

      return ListTile(
          enabled: enable,
          contentPadding: EdgeInsets.zero,
          title: title,
          subtitle: subtitle == null
              ? null
              : TextAtom(subtitle!, translate: translateSubtitle),
          leading: leading,
          trailing: trailing,
          onTap: onTap,
        );
  }
```